MiniSim/Menu.swift

```diff
@@ -125,6 +125,17 @@ class Menu: NSMenu {
                     NSPasteboard.general.copyToPasteboard(text: deviceID)
                     showNotification(title: "Device ID copied to clipboard!", body: deviceID)
                 }
+            case .deleteSim:
+                if let deviceID = device.ID {
+                    DispatchQueue.global().async { [self] in
+                        do {
+                            try deviceService.deleteSimulator(uuid: deviceID)
+                            showNotification(title: "Simulator deleted!", body: deviceID)
```

Comment:
We could fetch new devices after deletion, this way once the user opens menu this device will be already removed. 

After showing the notification, you can add a call to `getDevices()`